[PL-38538] Update delegate release notes notes, add ticket number, cross ref, fix typo (#1873)
harness/developer-hub: docs/platform/2_Delegates/manage-delegates/delegate-metrics.md

   :::

4. Select the exposed URL to access Grafana.

## Configure delegate resource threshold

You can set the delegate to reject new tasks if x% of memory is being consumed. You can then spin up new delegates when resources are above the threshold.

:::info note

> The resource_consumption_above_threshold metric is currently behind the feature flag `DELEGATE_ENABLE_DYNAMIC_HANDLING_OF_REQUEST`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.

Reviewer: ```suggestion
The `resource_consumption_above_threshold` metric is currently behind the feature flag `DELEGATE_ENABLE_DYNAMIC_HANDLING_OF_REQUEST`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
```

:::

> To configure the delegate resource threshold, do the following:

Reviewer: ```suggestion
To configure the delegate resource threshold, make the following changes to the delegate YAML file:
```

1. Set the `JAVA_OPTS` env variable.

   ```
   env:
       - name: JAVA_OPTS
         value: "-Xmx1536M"
   ```

2. Set the `DYNAMIC_REQUEST_HANDLING` env variable to `true` to enable dynamic task rejection.

   ```
   env:
      - name: DYNAMIC_REQUEST_HANDLING
        value: "true"
   ```

> 3. Set the `RESOURCE_USAGE_THRESHOLD` env variable to the threshold value of cpu/memory beyond which you want the delegate to reject new tasks.

Reviewer: ```suggestion
3. Set the `RESOURCE_USAGE_THRESHOLD` env variable to the cpu/memory threshold. When the threshold is exceeded, the delegate rejects new tasks.
```
